### ISSUE TITLE
Filenames with multiple dots lead to compile failure

### ISSUE THREAD

**Issue by MarkusAmshove** [casid/jte#9]
Hello!

I've been trying out `jte` and I really enjoy using it :-)

I'm trying to create templates for files but got a compile error for generated jte classes.

```
jte-classes\gg\jte\generated\subdirectory\Jtehelloworld\txtGenerated.java:2: error: '{' expected
public final class Jtehelloworld.txtGenerated {
```

I've pushed a [project to reproduce the behavior](https://github.com/MarkusAmshove/jterepro).

I've come to the conclusion, that the problem is the additional `.` in my template name: `helloworld.txt.jte`.

There are some workarounds for me, like naming it `helloworld.txt` or just `helloworld.jte` etc. but I thought maybe you're interested in supporting this scenario :-)

Other than that, awesome templating engine!

**Comment by casid** [comment]
Hi @MarkusAmshove,

thanks for the heads up, I'm so glad you're enjoying jte 👍 

And thanks for sharing the project to reproduce your finding! I just pushed a fix for it: https://github.com/casid/jte/commit/5edaf2a905319318a41f72f060af76e5db3a3968

Gonna submit a new release to maven central (`1.2.0`) within the next hour, including this bugfix.

**Comment by casid** [comment]
Released with `1.2.0`
